libraries/api/src/middleware/schema.js

```diff
@@ -45,12 +45,9 @@ const validateSchemas = ({ validator, absoluteSchemas, rootUrl, serviceName, ent
     // If input schema is defined we need to validate the input
     if (input) {
       if (!typeis(req, 'application/json')) {
```

Comment:
This may be out-of-scope, but we're not following the [type-is documented API](https://www.npmjs.com/package/type-is). The second argument should be a list:

```suggestion
      if (!typeis(req, ['application/json'])) {
```

or even ``['json']``. Peeking at the code, it [detects and converts to an array](https://github.com/jshttp/type-is/blob/c1f4388c71c8a01f79934e68f630ca4a15fffcd6/index.js#L55-L61), but I'm wary of using undocumented behavior.